e2e-tests/common/methods.py

```diff
@@ -33,3 +39,30 @@ def analyzer_supported_entities(data):
         f"{ANALYZER_BASE_URL}/supportedentities?{data}", headers=DEFAULT_HEADERS
     )
     return response.status_code, response.content
+
+
+def image_redactor(file_name: str, color_fill=None):
```

Comment:
would rename to 'redact' to match to the endpoint name, align with the other methods

Reply:
Done.